crates/bevy_pbr/src/deferred/pbr_deferred_functions.wgsl

```diff
@@ -80,9 +80,11 @@ fn pbr_input_from_deferred_gbuffer(frag_coord: vec4<f32>, gbuffer: vec4<u32>) ->
     let props = deferred_types::unpack_unorm3x4_plus_unorm_20_(gbuffer.b);
     // Bias to 0.5 since that's the value for almost all materials.
     pbr.material.reflectance = saturate(props.r - 0.03333333333);
+    pbr.specular_occlusion = props.b; // use diffuse occlusion as specular occlusion because we can't fit both in the g-buffer on webgl
```

Comment:
Shouldn't we just disable specular occlusion on webgl instead? I don't really know the impact of doing that tbh.

Reply:
i figured that would be a closer approximation than not occluding specular at all. this was discussed quite extensively around the original specular occlusion PR, but if not occluding specular at all on webGL is preferred then I don't really mind changing it

Reply:
I don't really have an opinion personally, it was mostly out of curiosity. I guess this is better than nothing yeah.

Reply:
I'd rather disable specular occlusion personally, and leave diffuse occlusion functional.

Reply:
As far as I know diffuse occlusion will not be affected by this

Reply:
What does this mean then?
> use diffuse occlusion as specular occlusion because we can't fit both in the g-buffer on webgl

Reply:
It uses the diffuse occlusion value for specular occlusion, but the diffuse occlusion value is also used for diffuse occlusion.  At least, that's my understanding.

Reply:
That understanding is correct. The needed data for specular occlusion calculation is not present here anymore, you need the non-multibounce raw SSAO output, and a couple other things that im pretty sure we do have, like NdotV type stuff.

Reply:
It's pretty debatable if it's even worth including occlusion in the deferred gbuffer in the first place. The gbuffer space is pretty precious, and baked AO is quite rare.